v4/source/introduction.yaml

```diff
@@ -214,23 +214,192 @@ tags:
 
       The list of Mattermost WebSocket events are:
       - added_to_team
+        ```
+        {
+          'event': 'added_to_team',
+          'seq': '<seq_id>'
+          'data': {
+            'team_id': '<team_id>',
+            'user_id': '<user_id>'
+          },
+          'broadcast': {
+            'omit_users': null,
+            'user_id': '<user_id>',
+            'channel_id': '<channel_id>',
+            'team_id': '<team_id>'
+          }
+        }
+        ```
       - authentication_challenge
-      - channel_converted
       - channel_created
+        ```
+        {
+          'event': 'channel_created',
+          'seq'  : '<seq_id>',
+          'data' : {
+            'channel_id': '<channel_id>',
+            'team_id'   : '<team_id>'
+          },
+          'broadcast': {
+            'omit_users': null,
+            'channel_id': '<channel_id>',
+            'team_id': '<team_id>',
+            'user_id': '<user_id>'
+          }
+        }
+        ```
       - channel_deleted
-      - channel_member_updated
+        ```
+        {
+          'event': 'channel_deleted',
+          'seq': '<seq_id>'
+          'data': {
+            'channel_id': '<channel_id>'
+           },
+          'broadcast': {
+            'omit_users': null,
+            'user_id': '<user_id>',
+            'channel_id': '<channel_id>',
+            'team_id': '<team_id>'
+          },
+
+          ```
       - channel_updated
+         ```
+         {
+           'event': 'channel_updated',
+           'seq': '<seq_id>'
+           'data': {
+             'channel': {
+               'id': '<channel_id>',
+               'create_at': '<timestamp>',
+               'update_at': '<timestamp>',
+               'delete_at': '<timestamp>',
+               'team_id': '<team_id>',
+               'type': 'O',
+               'display_name':'<channel_id>',
+               'name': '<channel_name>',
+               'header': 'header',
+               'purpose': '',
+               'last_post_at': '<timestamp>',
+               'total_msg_count': '<total_msg_count>',
+               'extra_update_at': '<timestamp>',
+               'creator_id': '<basic_user_id>'
```

Comment:
Curious, why `basic_user_id` instead of just `user_id`?

Reply:
See this - https://github.com/mattermost/mattermost-redux/blob/master/src/actions/websocket.test.js#L253. I picked from here.